### PR TITLE
ui: add target and target type in capture form

### PR DESCRIPTION
**Pull request by masco**

### CODE REVIEW

**Comment by masco** [comment]
run skydive-functional-hw-tests
run skydive-functional-tests-backend-elasticsearch

**Comment by masco** [comment]
run skydive-functional-hw-tests

**Comment by safchain** [comment]
run skydive-functional-hw-tests

File: statics/js/components/capture-form.js [hunk 1]
```diff
@@ -80,6 +80,16 @@ Vue.component('capture-form', {
               <label for="netflowTarget">NetFlow collector address, if empty the agent will be used</label>\
               <input id="netflowTarget" type="text" class="form-control input-sm" v-model="target"/>\
             </div>\
+            <div class="form-group" v-if="captureType != \'ovssflow\' && captureType != \'ovsnetflow\'">\
+              <label for="target">Target</label>\
```
Comment (safchain) [inline statics/js/components/capture-form.js]: It possible to specify a target for ovssflow and ovsnetflow but not the target type as we know if it is sflow or netflow. So the Target field should be outside of the if statement

**Comment by masco** [comment]
run skydive-compile-tests
run skydive-functional-tests-backend-elasticsearch
run skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run skydive-compile-tests
run skydive-functional-tests-backend-elasticsearch
run skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run skydive-compile-tests

**Comment by masco** [comment]
run skydive-compile-tests
run skydive-functional-tests-backend-elasticsearch
run skydive-functional-tests-backend-orientdb

**Pull request merged**